### DISCUSSION TITLE
is it possible to show video length (建议显示视频总时长）

### DISCUSSION THREAD

**Discussion by arroy2024** [QL-Win/QuickLook#1459] [Ideas]
When use quicklook to preview mp4 files, it shows current video position, but does not show how long it is, so I have to use other video player to open again. So suggest add this feature, thanks.

用Quicklook 预览视频时候，能看到当前播放到多少分、多少秒，但不显示总时长，所以非常不便，需要用其它播放器再次打开，建议显示视频总时长，谢谢。

**Comment by emako** [comment]
你需要点击“当前播放到多少分、多少秒”，会切换显示模式为“总时长”

**Reply by arroy2024** [reply]
感谢答复，不过是否增加一个设置，允许用户自行选择展示“总时长”或“当前播放到多少分和秒”呢？一方面，貌似大部分播放器都是默认显示总时长的，另一方面，当需要比较两个视频哪个是完整视频并删除其中一个的时候，总时长比较直观。

再次感谢。
